meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,221 @@
+
+When launching Meilisearch, use the following flag:
+```bash
+meilisearch --experimental-score-details
+```
+
+🗣️ Since this feature is experimental, we need your help to iterate on it! You are more than welcome to share your thoughts and your feedback on [this discussion](https://github.com/meilisearch/meilisearch/discussions/594).
+
+⚠️ The experimental feature API can break between Meilisearch versions. Be careful if using it in production.
+
+Done by @dureuill in #3771
+
+## Define field to search on at search time
+
+v1.3.0 introduces a new search parameter, `attributesToSearchOn`.
+
+`attributesToSearchOn` is a search parameter accepting an array of strings indicating one or more document attributes. Queries using `attributesToSearchOn` will restrict the search to the indicated attributes.
+
+Given the following dataset:
+```json
+{
+  "id": 0,
+  "name": "Our Wives Under the Sea",
+  "genre": ["horror", "scifi"],
+  "synopsis": "A woman returns to her wife transformed after a deep-sea adventure."
+},
+{
+  "id": 1,
+  "name": "A Strange and Stubborn Endurance",
+  "genre": ["adventure"],
+  "synopsis": "A man must overcome trauma and fight off a mysterious assassin."
+}
+```
+And the following query:
+```json
+{
+  "q": "adventure",
+  "attributesToSearchOn": ["genre"]
+}
+```
+Meilisearch will only return document 1.
+
+Both documents contain the word adventure, but `"attributesToSearchOn": ["genre"]` instructs Meilisearch to only consider results found on the genre field.
+
+Done by @ManyTheFish in (#3834)
+
+
+## Search for facet values
+Meilisearch v1.3.0 introduces a new endpoint  `POST /indexes/{index}/facet-search` that allows you to search for facet values. Only fields defined as `filterableAttributes` will be facet searchable.
+
+Done by @Kerollmops in (#3611)
+
+## Sort facet values by count
+By default, all facet values are sorted in ascending alphanumeric order. With the release of v1.3.0 you can change this using the `sortFacetValuesBy` property of the [`faceting` index settings](/reference/api/settings#faceting). This will allow you to sort facet by value count i.e. the genre with the highest number of results shows first.
+
+Done by @Kerollmops in (#3612)
+
+### Performance improvements
+
+* Use `RoaringBitmap::deserialize_unchecked_from` to reduce the deserialization time (#3788) @Kerollmops 
+* Remove the `docid_word_positions` database (#3819) @loiclec
+
+
+
+## Language support improvements
+
+* TBD charabia changes to be included
+
+## Other improvements
+* Improve the metrics (#3789) @irevoire
+
+
+# Fixes 🐞
+
+  * Changes the way words are counted in the word count DB (#3824) @dureuill 
+  * Fix addition deletion bug (#3670) @irevoire
+
+# Misc
+
+
+* Dependencies upgrade
+  * Bump actions/setup-go from 3 to 4 (#3805) @dependabot
+  * Bump Swatinem/rust-cache from 2.2.1 to 2.4.0 (#3803) @dependabot 
+  * Bump svenstaro/upload-release-action from 2.5.0 to 2.6.1 (#3804) @dependabot
```

Comment:
No need to put dependabot, not a human 😊

Reply:
do you still include the PRs in the changelog & just not mention dependabot with @ ?

Reply:
PRs are more important to mention than dependabot